apps/desktop/src/main/lib/agent-setup/templates/copilot-hook.template.sh

```
@@ -0,0 +1,47 @@
+#!/bin/bash
+{{MARKER}}
```

Shellcheck (0.11.0)

[error] You need a space after the '{'.

(SC1054)